python/examples/minimal_build/README.md

```diff
@@ -34,10 +34,21 @@ docker build -t arrow_fedora_minimal -f Dockerfile.fedora .
 Then build PyArrow with conda or pip, respectively:
 ```
 # With pip
-docker run --rm -t -i -v $PWD:/io arrow_fedora_minimal /io/build_venv.sh
+docker run --rm -t -i -v $PWD:/io -v $PWD/../../..:/arrow arrow_fedora_minimal /io/build_venv.sh
 
 # With conda
-docker run --rm -t -i -v $PWD:/io arrow_fedora_minimal /io/build_conda.sh
+docker run --rm -t -i -v $PWD:/io -v $PWD/../../..:/arrow arrow_fedora_minimal /io/build_conda.sh
+```
+
+Alternatively you can use [Docker Compose][docker-compose] to build and run using:
```

Comment:
For the reviewer, do you think it would be better to only document the Docker compose interface as we do on the CPP minimal build instead of having `docker`, `docker-compose` and `podman`? See:
https://github.com/apache/arrow/blob/master/cpp/examples/minimal_build/README.md#running-the-example

Reply:
I agree with the idea.

It seems that we can use `docker-compose` with `podman` by `dnf install podman-plugins` and `systemctl enable --now podman.socket`. So we can document `docker-compose` interface and how to use `docker-compose` with `podman`.